docs/kwik-format.md

```diff
@@ -0,0 +1,235 @@
+# Kwik format
+
+**NOTE: this format is still experimental and subject to future changes. Subscribe to the Klustaviewas mailing list to be kept up to date.**
+
+TODO: update this document.
+TODO: move descriptions of PRB/PRM in another file?
+
+## Quick info
+
+You can open the Kwik files in recent versions of MATLAB (they have [native support for HDF5](http://www.mathworks.fr/fr/help/matlab/hdf5.html)).
+
+To read the spike times and cluster numbers, type (`filename` is the `.kwik` file):
+
+```matlab
+hdf5read(filename, '/channel_groups/0/spikes/time_samples');
+hdf5read(filename, '/channel_groups/0/spikes/clusters/main');
+```
+
+The `channel_group` number is the shank number (indexing starts at 0).
+
+## Data files
+
+All files are in HDF5.
+
+  * The data are stored in the following files:
+
+      * the **KWIK** file is the main file, it contains:
+          * all metadata
+          * spike times
+          * clusters
+          * recording for each spike time
+          * probe-related information
+          * information about channels
+          * information about cluster groups
+          * events, event_types
+          * aesthetic information, user data, application data
+      * the **KWX** file contains the **spiking data**: features, masks, waveforms
+      * the **KWD** files contain the **raw/filtered recordings**
+
+  * Once spike sorting is finished, one can discard the KWX and KWD files and just keep the KWIK file for subsequent analysis (where spike sorting information like features, waveforms... are not necessary).
+
+  * All files contain a **version number** in `/` (`kwik_version` attribute), which is an integer equal to 2 now.
```

Comment:
and there are other changes too... so yes. Can you do it in phy-doc and also in `phy/io` in a new PR?